meta.yaml

```diff
@@ -1,13 +1,12 @@
 {% set name = "hub" %}
-{% set version = HUB_VERSION %}
+{% set version = "1.2.2" %}
```

Comment:
why is this set as the version?

Reply:
This is specified because later versions have issues with humbug.